### PR TITLE
fix: log errors, improved messages

### PR DESCRIPTION
**Pull request by devotox**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ class FtpServer extends EventEmitter {
       pasv_min: 1024,
       pasv_max: 65535,
       pasv_url: null,
+      public_pasv_url: null,
```
Comment (trs) [inline src/index.js]: I don't see the need for this. The `pasv_url` should be your public IP that is provided to the client.

Comment (devotox) [reply src/index.js]: I tried to set that but then it was not able to find passive ports using the public IP

Comment (trs) [reply src/index.js]: Sounds like [this](https://github.com/trs/ftp-srv/pull/155)? If so, that'll be fixed in [v4.0.1](https://github.com/trs/ftp-srv/releases/tag/v4.0.1)

Comment (devotox) [reply src/index.js]: you are correct

File: src/connector/passive.js [hunk 2]
```diff
@@ -79,6 +79,10 @@ class Passive extends Connector {
           }
         });
       });
+    })
+    .catch((error) => {
+      this.log.trace(error.message);
+      return this.setupServer();
```
Comment (trs) [inline src/connector/passive.js]: Is this to try again if it fails? What if it perpetually fails forever?

Comment (devotox) [reply src/connector/passive.js]: What would you do here then? Just remove this line totally?

Comment (trs) [reply src/connector/passive.js]: Yeah I would let it fail. Disconnect and/or respond with an error code to the client.

**Review by forstermatth — APPROVED** [review]
This seems like a pretty benign change that can be merged.

**Review by trs — APPROVED** [review]

**Review by Jl14Salvador — APPROVED** [review]

**Pull request merged**